package.json

```diff
@@ -21,7 +16,7 @@
         "test" : "tap test/*.js"
     },
     "dependencies" : {
-        "traverse" : "https://github.com/apiaryio/js-traverse/tarball/master",
+        "@apiaryio/traverse" : "^1.0.0",
```

Comment:
We might need to also add `.npmrc` file into the repository?

Reply:
Well only reason would be if we would install dependencies in this repo. For example for tests. Which we are not.